incubator/hnc/Makefile

```diff
@@ -155,7 +156,11 @@ manifests: controller-gen
 
 # Run go fmt against code
 fmt:
-	go fmt ${DIRS}
+	gofmt -w ${GOFMT_DIRS}
+
+# Check if fmt was ran
+check-fmt:
+	@test -z $(shell gofmt -l ${GOFMT_DIRS})
```

Comment:
Can you comment a bit more how this command works? Thanks!

Reply:
`gofmt -l` list files whose formatting differs from gofmt's.
`test -z`: True if the length of string is zero.

So if there are no files differing from `gofmt` command, `test` returns `True` otherwise fails.

Reply:
Sorry, I mean "comment in the makefile," not in the PR review :) This is to make life easier for future maintainers.

Reply:
I've added a more descriptive comment :)